Client/Frontend/Browser/FindInPageBar.swift

```diff
@@ -191,6 +181,17 @@ class FindInPageBar: UIView {
     static var retrieveSavedText: String? {
         return UserDefaults.standard.object(forKey: FindInPageBar.savedTextKey) as? String
     }
+
+    // MARK: - Theme Applicable
+    func applyTheme(theme: Theme) {
+        let colors = theme.colors
+        topBorder.backgroundColor = colors.layer5Hover
+        searchText.textColor = colors.layerRatingF
+        matchCountView.textColor = colors.actionSecondaryHover
```

Comment:
Me again, looks like this color should also be changed since it looks very pale in dark mode. We can maybe try `colors.actionSecondary` instead of that hover one.

Reply:
I re-added the screenshots, please let me know if I need to change anything else. Thank you!